pages/Configuring/Binds.md

```diff
@@ -113,60 +128,78 @@ e -> repeat, will repeat when held.
 m -> mouse, see below
 ```
 
-## Mouse Binds
+Example Usage:
+
+```ini
+# Example volume button that allows press and hold
+binde=, XF86AudioRaiseVolume, exec, wpctl set-volume @DEFAULT_AUDIO_SINK@ 5%+
+
+# Example volume button that will activate even while an input inhibitor is active
+bindl=, XF86AudioLowerVolume, exec, wpctl set-volume @DEFAULT_AUDIO_SINK@ 5%-
+
+# Start wofi on hold and kill it on release
+bindr=SUPER, SUPER_L, exec, pkill wofi || wofi
```

Comment:
is this even correct? `bindr` fires *only* on release? How would it open wofi on hold?

Reply:
Probably not? I don't know and I can't test it either, ask @fufexan to clarify